main.js

```diff
@@ -0,0 +1,70 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const form = document.querySelector('#form');
+const bookContainer = document.getElementById('book-container');
+// const addBtn = document.getElementById('addBtn');
+// let rmvBtn;
+
+// console.log(addBtn);
+
+const library = [];
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+// localStorage.setItem('books',library)
```

Comment:
- [ ] You need to add a function to preserve data with local storage

Reply:
Nice work is done here. Well addressed!! :champagne: